wpilib/wpilib/samplerobot.py

```diff
@@ -117,6 +121,7 @@ def robotMain(self):
                      do you.
         
         """
+
```

Comment:
You, sir, are making my linter cry.

Reply:
Believe me I'd *love* to format my files correctly, but I like Dustin merging my PRs and the two very often don't combine

Reply:
FWIW, PEP 8 and PEP 257 say to _not_ have a blank line after a docstring in a method.